applications/migrations/0046_change_application_aggregate_data_key_names.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.7 on 2021-05-20 07:47
+
+from django.db import migrations
+
+from applications.models import ApplicationAggregateData
+
+NEW_KEY_NAMES = ["applied_min_duration_total", "applied_reservations_total"]
```

Comment:
@kieferbonk please note that this changes the API for EXISTING application's aggregate data. 
So what was "min_duration_total" is now called "applied_min_duration_total" and "reservations_total" is now "applied_reservations_total"